server/src/main/java/org/apache/seata/server/console/impl/db/GlobalLockDBServiceImpl.java

```diff
@@ -115,6 +116,13 @@ public PageResult<GlobalLockVO> query(GlobalLockParam param) {
         return PageResult.success(list, count, param.getPageNum(), param.getPageSize());
     }
 
+    private String getOrderByCondition() {
```

Comment:
Why is there a sorted SQL injection here?

Reply:
ditto